rllib/core/optim/marl_optimizer.py

```diff
@@ -0,0 +1,77 @@
+from typing import Any, Dict, Mapping, Union, Type
+from ray.rllib.core.rl_module.marl_module import MultiAgentRLModule
+from ray.rllib.core.rl_module.rl_module import ModuleID
+from ray.rllib.core.optim.rl_optimizer import RLOptimizer
+from ray.rllib.policy.sample_batch import DEFAULT_POLICY_ID
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.typing import TensorType
+
+
+class MultiAgentRLOptimizer(RLOptimizer):
+    def __init__(self, rl_optimizers: Mapping[ModuleID, RLOptimizer] = None):
```

Comment:
You should make RLOptimizer constructor a simple one as well and use `from_config()` definition in the base class